packages/node_modules/@webex/internal-plugin-calendar/src/calendar.js

```diff
@@ -34,6 +37,32 @@ const Calendar = WebexPlugin.extend({
       qs: options
     })
       .then((res) => res.body.items);
+  },
+
+  /**
+   * Retrieves an array of meeting participants for the meeting id
+   * @param {String} id
+   * @returns {Promise} Resolves with an object of meeting participants
+   */
+  getParticipants(id) {
```

Comment:
This method needs tests to validate the implementation works as expected.

Reply:
@prsaroha you are still missing any sort of test coverage for this function. I know that they are used in the `list` function, and the test for it covers them, but if they are public functions, we need tests calling it specifically like an sdk user would.

Another option is that if this and `getNotes` is meant to only be internally used by the `list` function, you could mark these as `@private` in the jsdoc so that the documentation generator will not print them.

Reply:
I'll modify the tests I wrote to cover them, looks like I forgot to change them after making changes and writing new transforms

Reply:
Sounds good. You'll need two new `describe` blocks in the integration suite: one for `describe('#getParticipants')` and one for `describe('#getNotes')`.